InsertCoin/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Insert(coin)::42GameDevClub</title>
     <link rel="stylesheet" href="./styles/style.css" />
+    <link rel="stylesheet" href="./styles/slide.css" />
```

Comment:
you divide into two stylesheets which one is about slides, the other one is all about styles and adaptation for devices